### PR TITLE
Different types of Crates

### PR DESCRIPTION
**Pull request by ProfCube**
Such as different openings, were the chest will open and the item will hover over the chest.

### CODE REVIEW

File: config/Multy Crate.yml [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+        onOpenEffects: 'happyVillager, DRINK'
+        dormantEffects: ''
+    reward:
+        minimumRewards: 1
+        maximumRewards: 1
+        rewards:
+            - 'cmd:(/crate givekey %player% VoteCrate 1), chance:(0.5), noduplicate:()'
+            - 'cmd:(/crate givekey %player% CommonCrate 1), chance:(0.4), noduplicate:()'
+            - 'cmd:(/crate givekey %player% RareCrate 1), chance:(0.3), noduplicate:()'
+MMS:
```
Comment (imWillX) [inline config/Multy Crate.yml]: Is MMS Crate already included in the configs folder?

File: config/README.md [hunk 2]
```diff
@@ -6,3 +6,15 @@ Here are some example configurations.
 | -------------- |:-------------:|:----------------:|
 | SpawnEgg       | imWillX       |                  |
 | MMS            | ProfCube      | Mob Spawners     |
+| RandomKeyPack  | ProfCube      | Random Key       |
+
+
+# Premaid Configs
```
Comment (imWillX) [inline config/README.md]: I recommend this suggestion to correct some spelling errors and to maintain consistency.
```# Prebuilt Config Templates
Here are some pre-made configurations that could be used to bootstrap your server!
|       Names      |       Creator    |           Topic        |
|:----------------:|:----------------:|:----------------------:|
|   MultiCrates    |  ProfCube        |      8 Custom Crates   |
```

File: config/Multy Crate.yml (hunk 1 above)
Comment (imWillX) [inline config/Multy Crate.yml]: Ignore this. I didn't notice that the MultiCrate.yml was suppose to be fully prebuilt configuration.

Comment (ProfCube) [inline config/Multy Crate.yml]: Yes

Comment (ProfCube) [inline config/Multy Crate.yml]: i put it into one crate.yml

**Comment by imWillX** [comment]
Closing due to 1.13 and 2.0.0 config changes

**Pull request closed without merging**